VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionCommentsFragment.java

```diff
@@ -40,18 +42,15 @@
     private DiscussionComment discussionComment;
 
     @Inject
-    DiscussionAPI discussionAPI;
-
-    @Inject
-    DiscussionCommentsAdapter discussionCommentsAdapter;
-
-    @Inject
     Router router;
 
     @Inject
     Context context;
 
-    private static final Logger logger = new Logger(CourseDiscussionCommentsFragment.class.getName());
+    private DiscussionCommentsAdapter discussionCommentsAdapter;
+
+    @Nullable
```

Comment:
Hmm yes, upon further consideration, you're probably right here. I guess the analyzer can confirm whether it's initialized in the constructor.